test.sh

```diff
@@ -2,11 +2,13 @@
 
 set -eux
 
-docker run \
-    -v $PWD:/work \
-    -w /work \
-    ruby:3.1 \
-        bash -c "git config --global --add safe.directory /work; \
-                 gem install bundler --no-document; \
-                 bundle install; \
-                 bundle exec rspec"
+echo "==> Starting test.sh"
+
+echo "==> Docker Run"
+
+docker run --rm \
+    --volume $PWD:/work \
+    --workdir /work cyberark/ubuntu-ruby-builder \
+    bash -c 'git config --global --add safe.directory /work && bundle && bundle exec rspec' || :
```

Comment:
Do we wan't this to default to true if it fails?

Reply:
Removed || :